omod-1.10/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_10/OrderResource1_10.java

```diff
@@ -102,6 +108,43 @@ public DelegatingResourceDescription getRepresentationDescription(Representation
 		}
 	}
 	
+	@Override
+	public Model getCREATEModel(Representation rep) {
```

Comment:
Since GET, CREATE and UPDATE are actions that  I are being documented, captial cases indicating  the action makes sense. I understand the deviation from camelCase but this makes the code more readable. I am fine with this. @dkayiwa I leave the final decision upto you